### PR TITLE
[#IOPAE-77] Fix how handle sandboxFiscalCode

### PR DESCRIPTION
**Pull request by LoFrance**
This PR add `sandbox fiscal code` if we don't pass any `authorized_recipients` inside body payload or add it to the list of `authorized_recipients`.

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [x ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by iwoak** [comment]
@LoFrance lint it ;)

File: CreateService/handler.ts [hunk 1]
```diff
@@ -185,10 +204,12 @@ export function CreateServiceHandler(
...
-          // (sandboxFiscalCode as unknown) as FiscalCode,
-          Array.isArray(authorizedRecipients)
-            ? authorizedRecipients
-            : [authorizedRecipients],
+          O.isSome(maybeAuthorizedRecipients)
+            ? [
+                (sandboxFiscalCode as unknown) as FiscalCode,
+                ...maybeAuthorizedRecipients.value
+              ]
+            : [(sandboxFiscalCode as unknown) as FiscalCode],
```
Comment (balanza) [inline CreateService/handler.ts:212]: Alternatively:
```ts
[
  (sandboxFiscalCode as unknown) as FiscalCode,
   ...O.getOrElse([])
]
```

File: CreateService/handler.ts [hunk 2]
```diff
@@ -147,6 +148,21 @@ const createServiceTask = (
     200
   );
 
+export const getAuthorizedRecipientsFromPayload = (
+  servicePayload: ServicePayload
+): O.Option<null | ReadonlyArray<FiscalCode>> =>
```
Comment (balanza) [inline CreateService/handler.ts:153]: Why `null`? `Option`s are used to avoid null-ish values, if we may encounter a `null` maybe we are doing it wrong.

**Review by balanza — CHANGES_REQUESTED** [review]
Please avoid `Option<null>` objects as it leads to inconsistent state.

**Comment by LoFrance** [comment]
> Please avoid `Option<null>` objects as it leads to inconsistent state.

Yep, sorry it was a mistake at the beginning of writing the function's signature and I forget to remove it.

File: CreateService/handler.ts (hunk 2 above)
Comment (LoFrance) [reply CreateService/handler.ts]: Fix with cecbe9f

File: CreateService/handler.ts (hunk 1 above)
Comment (LoFrance) [reply CreateService/handler.ts]: Done with cecbe9f

**Review by iwoak — APPROVED** [review]
LGTM

**Review by balanza — APPROVED** [review]

**Pull request merged**